### ISSUE TITLE
Meta-issue: Characterize metrics more thoroughly

### ISSUE THREAD

**Issue by dgkf** [pharmaR/riskmetric#83]
Our metric proposals can always benefit from thoughtful discussion about the nuances of their implementation. Often even the most straightforward metrics have edge cases that make implementation for at least one source of package metadata more complicated than expected.

In particular, we have a few recent metric proposals that seem straightforward at face value, but could use some consideration before they could be tackled. Please consider participating in the discussion for these issues to help think through how they would be implemented for our various sources of package metadata (from source code, a local installation, a CRAN or Bioconductor package webpage, or GitHub or GitLab source code repository). 

- [**Metric for Size of code base**](https://github.com/pharmaR/riskmetric/issues/66)
- [**Metric for Release Rate**](https://github.com/pharmaR/riskmetric/issues/65)
- [**Metric for Website availability**](https://github.com/pharmaR/riskmetric/issues/64)

Perhaps we should come up with an issue tag like, "Spec Needed" or "Open Discussion" for metric proposals that still need a bit of characterization before we can expect someone to pull it off the shelf and implement it?

**Comment by elong0527** [comment]
* Create a new issue label to provide more details (e.g. "details needed")
* A discussion may be required to address those metrics that need details. 
* From programming perspective, specific the details of implementation.

**Comment by elong0527** [comment]
It might be challenge to guess a website if it is not described in the package.  How about we first collect URLs from the description file? 

For example, [ggplot2](https://cran.r-project.org/web/packages/ggplot2/index.html ) will extract two url:  

* http://ggplot2.tidyverse.org
* https://github.com/tidyverse/ggplot2
